privaterelay/templates/includes/alias-card.html

```diff
@@ -108,27 +124,96 @@
+      <div class="c-alias-block-settings {% if not user_profile.has_premium %}is-hidden{% endif %}">
+        <form action="{% url 'emails-index' %}" method="POST" class="c-alias-block-settings-form" data-alias-id="{{ alias.id }}" data-type="{{ alias_type }}">
+          <input type="hidden" name="method_override" value="PUT">
+          <input type="hidden" name="api_token" value="{{ user_profile.api_token }}">
+          <input type="hidden" name="{{ alias_type }}_address_id" value="{{ alias.id }}">
+          <div class="c-alias-block-settings-title">
+            {% ftlmsg 'profile-promo-email-blocking-title' %}
           </div>
-        </div>
-
-        <!-- created by date -->
-        <div class="c-alias-data created-by">
-          <img class="" src="/static/images/icon-calendar-gray.svg" alt="">
-          <div>
-            <span class="c-alias-data-label">{% ftlmsg 'profile-label-created' %}</span>
-            <span class="c-alias-data-value">{{ alias.created_at|date:"M j, Y" }}</span>
+          <div class="c-alias-block-settings-inputs">
+            <div class="c-alias-block-settings-input t-line-left t-forward">
+              <input type="radio" name="aliasBlockSettings{{ alias.id }}" id="forwardAll{{ alias.id }}" value="forwardAll" checked />
+              <label for="forwardAll{{ alias.id }}">
+                <img src="{% static '/images/umbrella-closed-mobile.svg' %}" alt="">
+                <div class="c-alias-block-settings-radio-button"></div>
+                <span>{% ftlmsg 'profile-promo-email-blocking-option-none' %}</span>
+              </label>
+            </div>
+            <div class="c-alias-block-settings-spacer"></div>
+            <div class="c-alias-block-settings-input t-line-center t-critical">
+              <input type="radio" name="aliasBlockSettings{{ alias.id }}" id="criticalOnly{{ alias.id }}" value="criticalOnly"/>
+              <label for="criticalOnly{{ alias.id }}">
+                <img src="{% static '/images/umbrella-semi-mobile.svg' %}" alt="">
+                <div class="c-alias-block-settings-radio-button"></div>
+                <span>{% ftlmsg 'profile-promo-email-blocking-option-promotionals' %}</span>
+              </label>
+            </div>
+            <div class="c-alias-block-settings-spacer"></div>
+            <div class="c-alias-block-settings-input t-line-right t-block">
+              <input type="radio" name="aliasBlockSettings{{ alias.id }}" id="blockAll{{ alias.id }}" value="blockAll"/>
+              <label for="blockAll{{ alias.id }}">
+                <img src="{% static '/images/umbrella-open-mobile.svg' %}" alt="">
+                <div class="c-alias-block-settings-radio-button"></div>
+                <span>{% ftlmsg 'profile-promo-email-blocking-option-all' %}</span>
+              </label>
+            </div>
           </div>
+        </form>
+        <div class="c-alias-block-settings-description">
+          <ul>
```

Comment:
Nit: I don't think a `<ul>` is appropriate here, given that only one `<li>` is ever visible at a time.

Reply:
Good call out. Can we discuss this further? I tossed it into an `<ul>` because there were three states to iterate through. What would make more semantic sense?

Reply:
Could be just a `<div>` I think, or perhaps an `<output>`, given that it's the result of setting something on an `<input>`.

Reply:
Revised to `output` in 70e6f6032fd8873d87b3b2b58adf5d0636a1453b